app/src/androidTest/java/ch/sdp/vibester/activity/AuthenticationActivityTest.kt

```diff
@@ -26,7 +26,7 @@ import kotlin.random.Random
  */
 @RunWith(AndroidJUnit4::class)
 class AuthenticationActivityTest {
-    private val SLEEP_TIME: Long = 5000
+    private val sleepTime: Long = 5000
```

Comment:
Should we use camel case for all the constants from now on?

Reply:
I will say yes but only when they are inside a class